tally_ho/libs/middleware/idle_timeout.py

```diff
@@ -11,9 +13,16 @@ def __init__(self, get_response):
     def __call__(self, request):
         if request.user.is_authenticated:
             last_visit = request.session.get('last_visit')
+            site_id = getattr(settings, "SITE_ID", None)
+            IDLE_TIMEOUT = None
 
-            if last_visit:
+            try:
+                siteinfo = SiteInfo.objects.get(site__pk=site_id)
+                IDLE_TIMEOUT = siteinfo.user_idle_timeout
```

Comment:
This is very nit-picky, and up to your discretion, but I prefer that this line go into the `else` part of the `try` block.

Reply:
Hmm interesting, does the `else` part get executed when the `try` block is executed successfully?

Reply:
Yes, you only get to the else when the try part is successful.